docs/src/reference/asciidoc/appendix/breaking.adoc

```diff
@@ -23,10 +23,32 @@ We've increased the `es.scroll.size` default from `50` to `1000`. If you deploy
 ES-hadoop in a low memory environment, consider lowering `es.scroll.size` to
 avoid issues.
 
-==== Support for Scala 2.10 has been removed
+==== Deprecations
+
+The following are deprecated and will be removed in a future release:
+
+===== Spark 1.x
```

Comment:
Rather than rely on the previous heading, I'd add "is deprecated" so this is a little more scannable.
```suggestion
==== Spark 1.x support is deprecated
```